config/apache/craft.conf

```diff
@@ -1,5 +1,6 @@
+# This number is tuned for craft instances running on t2.large ec2 instances (8GB RAM) running 256M PHP threads (the craft default)
+MaxRequestWorkers 25
```

Comment:
Note also that as they only have a couple of vCPU, a number more like 4 would be appropriate anyway (i.e. 2x 2 vCPU).

This is OK because most web apps will spend 1/3 - 1/2 time in DB / service calls - probably can't get more throughput with more workers competing for CPU time.

Reply:
Thought about this a bit more over lunch, my final answer is it makes sense to me to aim for 3 x 256 and fit it on micros.

Ensure scaling happens aggressively when CPU use goes up over 30-40% and combine with previously discussed oom / overcommit changes to minimise harm of edge case.

Reply:
is this recommendation for t3 or t2 micros?

Reply:
I would say we should set the workers to run on micros but I would not actually change the instance types without further review.

Reply:
It is fine for either, but we shouldn't really bulk migrate until reservations expire and new ones are set.